core/controlplane/config/encrypted_assets.go

```diff
@@ -558,22 +564,28 @@ func ReadOrEncryptAssets(dirname string, manageCertificates bool, caKeyRequiredO
 		if file.readEncrypted {
 			data, err := encryptor.EncryptedCredentialFromPath(path, file.defaultValue)
 			if err != nil {
-				return nil, fmt.Errorf("Error encrypting %s: %v", path, err)
+				return nil, fmt.Errorf("error encrypting %s: %v", path, err)
 			}
 
 			*file.data = *data
 			if err := data.Persist(); err != nil {
-				return nil, fmt.Errorf("Error persisting %s: %v", path, err)
+				return nil, fmt.Errorf("error persisting %s: %v", path, err)
 			}
 		} else {
 			raw, err := RawCredentialFileFromPath(path, file.defaultValue)
 			if err != nil {
-				return nil, fmt.Errorf("Error reading credential file %s: %v", path, err)
+				return nil, fmt.Errorf("error reading credential file %s: %v", path, err)
 			}
 			if file.expiryCheck {
-				if err := tlsutil.CheckAllCertsValid(path, raw.content); err != nil {
+				certs, err := tlsutil.ToCertificates(raw.content)
```

Comment:
I prefer `tlsutil.CertificatesFromBytes` to make it even more explicit about in/out.
An alternative would be `certs.FromBytes` if we can safely extract a dedicated package `certs` out of the `tlsutil`. When we do many things in a single pkg named *util, I tend to split it into meaningful packages.